jooby/src/main/java/org/jooby/handlers/AssetHandler.java

```diff
@@ -253,9 +264,9 @@ private void doHandle(final Request req, final Response rsp, final Asset asset)
     }
 
     // cache max-age
-    if (maxAge > 0) {
-      rsp.header("Cache-Control", "max-age=" + maxAge);
-    }
+    maxAgeOpt.ifPresent(d -> {
```

Comment:
There is an ```assets.cache.maxAge = -1``` in ```jooby.conf```...think Assets.java use this and there we are probably setting a cache-control header with max-age= -1... not sure

Anyway, I'm close to release 1.0.0.CR3 and want to get all these in. Let me know when is done so I can merge this.

Thanks